config.json

```diff
@@ -1033,7 +1033,12 @@
       "core": false,
       "difficulty": 9,
       "slug": "pythagorean-triplet",
-      "topics": null,
+      "topics": [
+        "integer",
+	"lists",
+	"logic",
+	"mathematics"
```

Comment:
The indentation for `integer` and the closing square bracket is now correct :) But the other topics should be in  line with `integer` which they're not at the moment. Maybe opening the file in Intellij or Sublime text would help? I don't use Notepad++ much so I don't know much about it, but it does seem to be doing something strange to your spacing :P